### PR TITLE
Refactored styling of product page (#621)

### LINKED ISSUE #621: Simplify styling of product page
## What is the motivation for adding/enhancing this feature?
Refactor product page styling 

## What are the acceptance criteria?

- [ ] ...

## Can you complete this feature request by yourself?

- [X] YES
- [ ] NO

### PR DESCRIPTION
**Pull request by azkyakhan**
### Related Issues
#621 

Closes #621 

### Short Description and Why It's Useful
Simplified styling of product page

### Screenshots of Visual Changes before/after (If There Are Any)
product-page-desktop-before
product-page-desktop-after
product-page-mobile-before
product-page-mobile-after

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [X] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: components/molecules/m-product-additional-info.vue [hunk 1]
```diff
@@ -117,10 +120,6 @@ export default {
   }
 }
 
-[itemprop="description"] > *:first-child {
```
Comment (dt2patel) [inline components/molecules/m-product-additional-info.vue:120]: what was this CSS doing? what changes happen by removing it?

File: components/organisms/o-product-details.vue [hunk 2]
```diff
@@ -146,42 +151,35 @@ export default {
...
+
 @import "~@storefront-ui/shared/styles/helpers/breakpoints";
 
-.product {
-  @include for-desktop {
-    display: flex;
+.o-product-details {
+  @include for-desktop{
+    display: grid;
+    grid-template-columns: 1fr 1fr;
```
Comment (dt2patel) [inline components/organisms/o-product-details.vue:162]: how does this behave when the view port grows and shrinks? Does the product info section shrink or does it get cutoff?

File: components/organisms/o-product-details.vue [hunk 3]
```diff
@@ -146,42 +151,35 @@ export default {
...
-  &__guide,
-  &__compare,
-  &__save {
+}
+
+.secondary-action {
     display: block;
     margin: var(--spacer-xl) 0 var(--spacer-base) auto;
-  }
+
```
Comment (dt2patel) [inline components/organisms/o-product-details.vue:178]: Is this an extra space?

File: components/molecules/m-product-additional-info.vue (hunk 1 above)
Comment (azkyakhan) [reply components/molecules/m-product-additional-info.vue:120]: Description has 2 paragraphs so according to this CSS margin-top of first paragraph i.e. the first child should be 0. 
But I checked this style and margin wasn't set to 0. It was 16px only. So I decided to remove it.

itemdesciption

itemdescription2

File: components/organisms/o-product-details.vue (hunk 2 above)
Comment (azkyakhan) [reply components/organisms/o-product-details.vue:162]: It behaves some what the same way with grid also. Product info is shrinking.

product-info-before
product-info-after

File: components/organisms/o-product-details.vue (hunk 3 above)
Comment (azkyakhan) [reply components/organisms/o-product-details.vue]: Yes I have fixed it and updated the PR.

**Review by dt2patel — APPROVED** [review]
Looks good!

**Pull request merged**